plugin/trino-exchange/src/test/java/io/trino/plugin/exchange/containers/MinioStorage.java

```diff
@@ -0,0 +1,97 @@
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.exchange.containers;
+
+import com.google.common.collect.ImmutableMap;
+import io.trino.testing.containers.Minio;
+import org.testcontainers.containers.Network;
+import software.amazon.awssdk.auth.credentials.AwsBasicCredentials;
+import software.amazon.awssdk.auth.credentials.StaticCredentialsProvider;
+import software.amazon.awssdk.services.s3.S3Client;
+import software.amazon.awssdk.services.s3.model.CreateBucketRequest;
+
+import java.net.URI;
+import java.util.Map;
+
+import static java.util.Objects.requireNonNull;
+import static org.testcontainers.containers.Network.newNetwork;
+import static software.amazon.awssdk.regions.Region.US_EAST_1;
+
+public class MinioStorage
+        implements AutoCloseable
+{
+    public static final String ACCESS_KEY = "accesskey";
+    public static final String SECRET_KEY = "secretkey";
+
+    private final String bucketName;
+    private final Network network;
+    private final Minio minio;
+
+    public MinioStorage(String bucketName)
+    {
+        this.bucketName = requireNonNull(bucketName, "bucketName is null");
+        this.network = newNetwork();
+        this.minio = Minio.builder()
+                .withNetwork(network)
+                .withEnvVars(ImmutableMap.<String, String>builder()
+                        .put("MINIO_ACCESS_KEY", ACCESS_KEY)
+                        .put("MINIO_SECRET_KEY", SECRET_KEY)
+                        .build())
+                .build();
+    }
+
+    public void start()
+    {
+        minio.start();
+        S3Client s3Client = S3Client.builder()
+                .endpointOverride(URI.create("http://localhost:" + minio.getMinioApiEndpoint().getPort()))
+                .credentialsProvider(StaticCredentialsProvider.create(AwsBasicCredentials.create(ACCESS_KEY, SECRET_KEY)))
+                .region(US_EAST_1)
+                .build();
+        CreateBucketRequest createBucketRequest = CreateBucketRequest.builder()
+                .bucket(bucketName)
+                .build();
+        s3Client.createBucket(createBucketRequest);
+    }
+
+    public Minio getMinio()
+    {
+        return minio;
+    }
+
+    public String getBucketName()
+    {
+        return bucketName;
+    }
+
+    @Override
+    public void close()
+            throws Exception
+    {
+        network.close();
+        minio.close();
+    }
+
+    public static Map<String, String> getExchangeManagerProperties(MinioStorage minioStorage)
+    {
+        return new ImmutableMap.Builder<String, String>()
+                .put("exchange.base-directory", "s3n://" + minioStorage.getBucketName())
+                .put("exchange.s3.aws-access-key", MinioStorage.ACCESS_KEY)
+                .put("exchange.s3.aws-secret-key", MinioStorage.SECRET_KEY)
+                .put("exchange.s3.region", "us-east-1")
```

Comment:
Yes, otherwise an exception will be thrown:
```
SdkClientException: Unable to load region from any of the providers in the chain DefaultAwsRegionProviderChain
```

Reply:
Interesting. Maybe we should have it as a default?